src/routes/youtube/+page.svelte

```diff
@@ -4,81 +4,82 @@
 
   export let data: PageData;
 
-  let videoData = data.payload.videoData;
-  let channelData = data.payload.channelData;
+  const videoData = data.payload.videoData;
+  const channelData = data.payload.channelData;
   let displayedVideos = videoData;
 
   let rerender: boolean = false;
-
-  // Sort channel data by subcount (bubbling Climate Town to the top)
-  function channelSort(a: any, b: any) {
+  
+  interface Channel {
+    channelId: string;
+    channelSubCount: number;
+  };
+  interface ChannelFilterItem { 
+    channelId: string;
+    active: boolean;
+  };
+  interface Video {
+    channelId: string;
+    [key: string]: any;
+  };
+
+  function sortChannelBySubCount(a: Channel, b: Channel) {
     const ClimateTownChannelId = "UCuVLG9pThvBABcYCm7pkNkA";
+    
     if (a.channelId === ClimateTownChannelId) {
       return -1;
     }
     if (b.channelId === ClimateTownChannelId) {
       return 1;
     }
 
-    // Normal sort
```

Comment:
Please keep this comment (even if its just for me and my Typescript noobiness :) )